### PR TITLE
Add guide for running RepEnrich2.

### PR DESCRIPTION
**Pull request by roryk**
RepEnrich2 is a way to look at enrichment of repetitive elements. It makes me realize we should release more tools, the code base is a little bit of a mess so I included a link to a fork I made that  had some quality of life improvements that make it a lot more pleasant to run. I wasn't sure what tag to put for this, so I left it blank.

### CODE REVIEW

File: research/rnaseq/RepEnrich2_guide.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: How to run repeat enrichment analysis
+description: This guide shows how to run RepEnrich2
+category: research
+subcategory: rnaseq
+tags: []
```
Comment (lpantano) [inline research/rnaseq/RepEnrich2_guide.md]: `annotation` will be good

Comment (roryk) [reply research/rnaseq/RepEnrich2_guide.md]: You're still here! <3 <3 <3

Comment (lpantano) [reply research/rnaseq/RepEnrich2_guide.md]: I am a ghost, but still here :)

Comment (roryk) [reply research/rnaseq/RepEnrich2_guide.md]: 👻 👻 👻 👻 👻 👻

**Pull request merged**